### ISSUE TITLE
[Update Request]: LilyPond.LilyPond 2.25.22

### ISSUE THREAD

**Issue by luzpaz** [microsoft/winget-pkgs#114201]
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

LilyPond.LilyPond

### Package Version

2.25.22

### Please describe the changes you would like to see

https://gitlab.com/lilypond/lilypond/-/releases/v2.25.22

**Comment by CoolPlayLin** [comment]
Hi @luzpaz 
The latest version of this software is exe, but, it appears I haven't found the exe file in the URLs you provided. And neither did I in http://lilypond.org/download/binaries/mingw/ . Is this expected?

![image](https://github.com/microsoft/winget-pkgs/assets/96709586/0b2ac3a3-8afb-455a-aea9-536e796ceceb)
![image](https://github.com/microsoft/winget-pkgs/assets/96709586/57a579ef-4fea-43f6-a5d5-8e8ab4bd34e1)

**Comment by lemzwerg** [comment]
Yes, it is expected; self-extracting archives have been replaced with ZIP archives.

https://lilypond.org/doc/v2.25/Documentation/learning/graphical-setup-under-windows

**Comment by luzpaz** [comment]
Is there anything blocking this ?

**Comment by luzpaz** [comment]
Looks like #123833 failed. Perhaps trying latest https://gitlab.com/lilypond/lilypond/-/releases/v2.25.12 ?

**Comment by luzpaz** [comment]
bump

**Comment by luzpaz** [comment]
https://gitlab.com/lilypond/lilypond/-/releases/v2.25.13 released

**Comment by Trenly** [comment]
[Policy] Unblocked; Will require `ManifestVersion: 1.9.0` with `ArchiveBinariesDependOnPath: true`

**Comment by luzpaz** [comment]
2.25.22 is latest development release (http://lilypond.org/development.html)
